### PR TITLE
Can't resize server pools

### PR DESCRIPTION
**Pull request by feorlen**
Explicitly note in the concepts page that you can't change the size of a server pool after it's created.

Not staged

### CODE REVIEW

File: source/operations/concepts.rst [hunk 1]
```diff
@@ -107,6 +107,7 @@ MinIO :ref:`distributed deployments <minio-mnmd>` support expansion and decommis
 
 Expansion consists of adding one or more :ref:`server pools <minio-intro-server-pool>` to an existing deployment.
 Each server pool consists of dedicated nodes and storage that contribute to the overall capacity of the deployment.
+The size of an individual server pool is fixed at the time it is created.
```
Comment (feorlen) [inline source/operations/concepts.rst:110]: Now that I look at this again, could be reworded.

Comment (djwfyi) [inline source/operations/concepts.rst]: Just trying to get it out of passive voice.

```suggestion
Once set, a server pool's size cannot change.
```

**Review by djwfyi — CHANGES_REQUESTED** [review]
A wording suggestion to simplify and remove the passive voices.

File: source/operations/concepts.rst (hunk 1 above)
Comment (feorlen) [reply source/operations/concepts.rst]: I'm very confused at what GitHub just did, your suggestion is referencing the previous version of the text.

**Review by djwfyi — APPROVED** [review]
LGTM!

File: source/operations/concepts.rst (hunk 1 above)
Comment (djwfyi) [reply source/operations/concepts.rst]: Odd, very odd

**Pull request merged**